WooCommerce/Classes/ViewRelated/Products/Media/MediaPickingCoordinator.swift

```diff
@@ -0,0 +1,93 @@
+import UIKit
+
+/// Prepares the alert controller that will be presented when trying to add media to a site.
+///
+final class MediaPickingCoordinator {
+    private lazy var cameraCapture: CameraCaptureCoordinator = {
+        return CameraCaptureCoordinator(onCompletion: onCameraCaptureCompletion)
+    }()
+
+    private lazy var deviceMediaLibraryPicker: DeviceMediaLibraryPicker = {
+        return DeviceMediaLibraryPicker(onCompletion: onDeviceMediaLibraryPickerCompletion)
+    }()
+
+    private let onCameraCaptureCompletion: CameraCaptureCoordinator.Completion
+    private let onDeviceMediaLibraryPickerCompletion: DeviceMediaLibraryPicker.Completion
+
+    init(onCameraCaptureCompletion: @escaping CameraCaptureCoordinator.Completion,
+         onDeviceMediaLibraryPickerCompletion: @escaping DeviceMediaLibraryPicker.Completion) {
+        self.onCameraCaptureCompletion = onCameraCaptureCompletion
+        self.onDeviceMediaLibraryPickerCompletion = onDeviceMediaLibraryPickerCompletion
+    }
+
+    func present(context: MediaPickingContext) {
```

Comment:
nit: I'd try to remove some carriage return inside this method (for example all the spaces between every addAction)

Reply:
TIL something new 😮 thanks @mindgraffiti!